pkg/providers/aws/provider_blobstorage.go

```diff
@@ -35,10 +37,11 @@ const (
 	blobstorageProviderName    = "aws-s3"
 	defaultAwsBucketNameLength = 40
 	// default create options
-	dataBucketName          = "bucketName"
-	dataBucketRegion        = "bucketRegion"
-	dataCredentialKeyID     = "credentialKeyID"
-	dataCredentialSecretKey = "credentialSecretKey"
+	dataBucketName             = "bucketName"
+	dataBucketRegion           = "bucketRegion"
+	dataCredentialKeyID        = "credentialKeyID"
+	dataCredentialSecretKey    = "credentialSecretKey"
+	defaultForceBucketDeletion = true
```

Comment:
This should be defaulting to `false`, right?

Reply:
Yeah, that was from my own testing, good catch